docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -6,7 +6,7 @@ title: Telemetry
 
 Kyma's observability functionality focuses on the pre-integrated collection of telemetry data from the users' workloads, and making that data available for further analysis and storage in backends of any kind and location. The Telemetry component provides the configurable collection and shipment components, and with that, separates them explicitly from the storage and analysis in a specific backend system. You can still choose to install lightweight in-cluster backends with dedicated observability components.
 
-To support the logging domain, the Telemetry component provides a log collector, [Fluent Bit](https://fluentbit.io/). You can configure the log collector with external systems using runtime configuration with a dedicated Kubernetes API (CRD) named `LogPipeline`. Using the LogPipeline's HTTP output, you can natively integrate with vendors such as [VMWare](https://medium.com/@shrishs/log-forwarding-from-fluent-bit-to-vrealizeloginsightcloud-9eeb14b40276), or with any vendor using a [Fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa). Furthermore, you can run the collector in an [unsupported mode](#unsupported-mode), leveraging the full output options of Fluent Bit. Additionally, you can always bring your own log collector if you need advanced configuration options.
+To support the logging domain, the Telemetry component provides a log collector, [Fluent Bit](https://fluentbit.io/). You can configure the log collector with external systems using runtime configuration with a dedicated Kubernetes API (CRD) named `LogPipeline`. Using the LogPipeline's HTTP output, you can natively integrate with vendors or with any vendor using a [Fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa). Furthermore, you can run the collector in an [unsupported mode](#unsupported-mode), leveraging the full output options of Fluent Bit. Additionally, you can always bring your own log collector if you need advanced configuration options.
```

Comment:
```suggestion
To support the logging domain, the Telemetry component provides a log collector, [Fluent Bit](https://fluentbit.io/). You can configure the log collector with external systems using runtime configuration with a dedicated Kubernetes API (CRD) named `LogPipeline`. Using the LogPipeline's HTTP output, you can natively integrate with vendors which are supporting this output, or with any vendor using a [Fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa). Furthermore, you can run the collector in an [unsupported mode](#unsupported-mode), leveraging the full output options of Fluent Bit. Additionally, you can always bring your own log collector if you need advanced configuration options.
```

Reply:
```suggestion
To support the logging domain, the Telemetry component provides a log collector, [Fluent Bit](https://fluentbit.io/). You can configure the log collector with external systems using runtime configuration with a dedicated Kubernetes API (CRD) named `LogPipeline`. Using the LogPipeline's HTTP output, you can natively integrate with vendors which support this output, or with any vendor using a [Fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa). Furthermore, you can run the collector in an [unsupported mode](#unsupported-mode), leveraging the full output options of Fluent Bit. Additionally, you can always bring your own log collector if you need advanced configuration options.
```
This one will also do. Just a small language correction and it's good to go :)